float64 distance
int64 ECHO
int64 TRIG
